packages/semantic-release-wildcards/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "semantic-release-wildcards",
+  "version": "0.0.0",
+  "bugs": "https://github.com/IBM/test-automation/issues",
+  "description": "Monorepo containing automated testing and tools for Carbon Shared Tests and Exported Tests",
+  "main": "src/index.js",
```

Comment:
Based on our repo architecture convo shouldn't these files be under `lib` not `src`?